<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dSelfTest</class>
 <widget class="QDialog" name="dSelfTest">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Самоконтроль</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>449</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pbStartTest">
      <property name="text">
       <string>Начать</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pbStopTest">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Прервать</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pbClose">
      <property name="text">
       <string>Закрыть</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>221</y>
     <width>651</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>651</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Режим проверки</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>160</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="cbPVD1Test">
       <property name="text">
        <string>Проверка ПВД 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbPVD2Test">
       <property name="text">
        <string>Проверка ПВД 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbMAG1Test">
       <property name="text">
        <string>Проверка МАГ1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbMAG2Test">
       <property name="text">
        <string>Проверка МАГ2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>20</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Время проверки:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbCheckTime">
       <property name="maximum">
        <number>600</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>160</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="cbMSS1Test">
       <property name="text">
        <string>Проверка МСС1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbMSS2Test">
       <property name="text">
        <string>Проверка МСС2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbMSS3Test">
       <property name="text">
        <string>Проверка МСС3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbMSS4Test">
       <property name="text">
        <string>Проверка МСС4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbMSS5Test">
       <property name="text">
        <string>Проверка МСС5</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>160</y>
     <width>651</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Ход проверки</string>
   </property>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>20</y>
      <width>641</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
